plugin-taskbar/lxqttaskbar.h

```diff
@@ -65,16 +65,17 @@ class LxQtTaskBar : public QFrame
 
     void realign();
 
-    Qt::ToolButtonStyle buttonStyle() { return mButtonStyle; }
-    int buttonWidth() { return mButtonWidth; }
-    bool closeOnMiddleClick() { return mCloseOnMiddleClick; }
-    bool raiseOnCurrentDesktop() { return mRaiseOnCurrentDesktop; }
-    bool isShowOnlyCurrentDesktopTasks() { return mShowOnlyCurrentDesktopTasks; }
-    bool isShowOnlyCurrentScreenTasks() { return mShowOnlyCurrentScreenTasks; }
-    bool isShowOnlyMinimizedTasks() { return mShowOnlyMinimizedTasks; }
-    bool isAutoRotate() { return mAutoRotate; }
-    bool isGroupingEnabled() { return mGroupingEnabled; }
-    bool isShowGroupOnHover() { return mShowGroupOnHover; }
+    Qt::ToolButtonStyle buttonStyle() const throw () { return mButtonStyle; }
```

Comment:
Well, no problem to use that. I just tend to follow the pattern, and (as you said) we don't use it.